.github/workflows/test.yml

```diff
@@ -28,32 +28,15 @@ jobs:
         python-version: ${{ matrix.python-version }}
 
     ### Caching of pip downloads and local wheel builds
+    - name: Get pip cache dir
+      id: pip-cache
+      run: |
```

Comment:
Nice change to remove some annoying duplication here. Thanks!

BTW this could be a single line for `run` without the `|` multiline indication.